docs/metrics.md

```diff
@@ -223,8 +220,6 @@ EOF
 
 Create a *RoleBinding* for Binding the *Role* to Prometheus' monitoring *ServiceAccount*:
 
-> Note: for *OCP* *subjects[0].namespace* should be `openshift-monitoring`.
```

Comment:
Is this changed?

Reply:
yes, besides the _ServiceMonitors_ which are included in this PR, all other resources already exist in _OCP_.
so I think, this doc will be used primarily by _K8S_ _OCM_ users, so this comment seems redundant to me.
I can bring it back if you think it's helpful.

Reply:
on second thought...
this will be useful for users who run _OCM_ on _OCP_.
so maybe we should indeed bring back...

your call.
:-)

Reply:
Yeah I think we should bring it back if there are other use cases using this note

Reply:
done.